core/vm/jump_table.go

```diff
@@ -1021,13 +1021,8 @@ func newFrontierInstructionSet() JumpTable {
 			memorySize: memoryReturn,
 			halts:      true,
 		},
-		SELFDESTRUCT: {
-			execute:    opSuicide,
-			dynamicGas: gasSelfdestruct,
-			minStack:   minStack(1, 0),
-			maxStack:   maxStack(1, 0),
-			halts:      true,
-			writes:     true,
-		},
+		// SELFDESTRUCT is disabled in Scroll.
```

Comment:
ditto

Reply:
Same here, I'd prefer to disable this opcode more explicitly, if this only costs us this one line.